### ISSUE TITLE
"Filter values" input box not working in Internet Explorer 11

### ISSUE THREAD

**Issue by co-operate** [nicolaskruchten/pivottable#850]
Just a heads up. Filtering values in the attribute bubble does, however, work in Microsoft Edge, Firefox  and Chrome.

PS: And once more THANK YOU to Nicolas and all other contributors for this really outstanding piece of software!

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! I'm not sure when this stopped working but I'll try to get it fixed soon!

**Comment by co-operate** [comment]
Nicolas,

Sorry, I noticed this is a duplicate of issue #640. 
The solution proposed works, there is to polyfill ".startsWith“:

```
if (!String.prototype.startsWith) {
	String.prototype.startsWith = function(search, pos) {
		return this.substr(!pos || pos < 0 ? 0 : +pos, search.length) === search;
	};
};
```
Alternatively, there is a more robust polyfill by Mathias Bynens https://github.com/mathiasbynens/String.prototype.startsWith

**Comment by nicolaskruchten** [comment]
Oh, I didn't dig into that older issue as much as I should have :(

This is actually really easy to fix, I don't even need a polyfill, I'll just remove the calls to `startsWith`.

**Comment by co-operate** [comment]
Even better :-)

**Comment by nicolaskruchten** [comment]
Let's leave this open until I've released a fix, though, please :)

**Comment by nicolaskruchten** [comment]
Fix released in version 2.19.0 :)
